docs/personal-data.md

```diff
@@ -194,14 +200,44 @@ A list of all the [Identity events]((../src/Surfnet/Stepup/Identity/Event/) in s
 - identity_institution
 - name_id
 - Forgettable: email
-- Forgettable: commonName
+- Forgettable: common_name
+
+[RegistrationAuthorityRetractedForInstitutionEvent](../src/Surfnet/Stepup/Identity/Event/RegistrationAuthorityRetractedForInstitutionEvent.php)
+- identity_id
+- identity_institution
+- name_id
+- ra_institution
+- Forgettable: email
+- Forgettable: common_name
+
+[SecondFactorMigratedEvent](../src/Surfnet/Stepup/Identity/Event/SecondFactorMigratedEvent.php)
+- identity_id
+- source_institution
+- target_name_id
+- identity_institution
+- second_factor_id
+- new_second_factor_id
+- second_factor_type
+- preferred_locale
+- Forgettable: second_factor_identifier
+- Forgettable: common_name
+- Forgettable: email
+
+[SecondFactorMigratedToEvent](../src/Surfnet/Stepup/Identity/Event/SecondFactorMigratedToEvent.php)
+- identity_id
+- identity_institution
+- second_factor_id
+- target_institution
+- target_second_factor_id
+- second_factor_type
+- Forgettable: second_factor_identifier
```

Comment:
Thanks for doing my job! Last year I implemented the token move feature. But forgot to add the new events to the docs.
:bow: